api/graphql/schema.graphql

```diff
@@ -1257,6 +1257,7 @@ input AwardExperienceInput {
 
 input UpdateExperienceSkills {
   connect: [ConnectExperienceSkills!] @resolveWith(method: "connectSkills")
+  update: [ConnectExperienceSkills!] @resolveWith(method: "connectSkills")
```

Comment:
Could you please leave a comment for future explorers on why this is so weird?  😅

Reply:
Added in [document aliased input item](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8726/commits/638cc68581f9032d09cbb6129fc3ba71c09db5e9)